reactioncommerce/reaction-next-starterkit
# feat: Cart Popover component

Resolves #166 
Impact: **minor**
Type: **feature**

## Issue
- Create a popover component which displays the most recent item added to the cart.
- This component should display a single Cart Item and a button with a link to the checkout page.
- This component should be added to the Storefront starter-kit
- This component should only be displayed on mobile devices

## Solution
Create the above mentioned Cart Popover

## Breaking changes
None

## Notes
- This component does not yet have any live data. This should be added in #159. For the time being, there is a `const` with static data for testing purposes. This same data will be passed in via a prop when it's available.
- I don't feel like this looks as good as it can when it's on a screen greater thatn 600px. I think we should re-visit the breakpoints, and use the future modal (no ticket yet) or the mini-cart (#128) to display instead of this component at the 600px breakpoint.

## Testing
1. Open the site on a mobile sized device (less than 960px)
1. Add an item to the cart
1. See that the cart pop-up displays for 3 seconds
1. Add another item to cart
1. Move mouse into display when visible
1. See the pop up doesn't disappear
